javascript-loops-practice/filter-out-strings.js

```diff
@@ -0,0 +1,16 @@
+/* exported filterOutStrings */
+function filterOutStrings(values) {
+  const filteredStrings = [];
+  for (let i = 0; i < values.length; i++) {
+    const typeOfValue = typeof values[i];
+
+    if (
+      typeOfValue === 'number' ||
+      typeOfValue === 'null' ||
+      typeOfValue === 'object'
+    ) {
+      filteredStrings.push(values[i]);
+    }
+  }
```

Comment:
Similar thing to my last comment. Rather than checking all of the other potential types, you can just use the logical not operator to check if it's not a string